app.js

```diff
@@ -1,21 +1,29 @@
 'use strict';
 
 var userResponse = '';
+var favoriteNum = 11;
+var userGuess;
+var notGuessed = true;
+var countriesVisited = ['canada', 'mexico', 'england', 'france', 'italy', 'spain', 'greece', 'croatia'];
+var numQuestions = 0;
+var numCorrect = 0;
```

Comment:
It's a very good idea to put your variables at the top of the document. It makes it easier to change or find a variable. otherwise it's hard to find where it's declared.